composer.json

```diff
@@ -62,5 +63,11 @@
 				"./tools/pre-commit.sh"
 			]
 		}
-	}
+    },
+    "autoload": {
+        "psr-4": {
```

Comment:
The *only* reason we can't adopt PSR4 autoloading is that we're using `.inc.php` extensions rather than `.php`. I'd definitely like to move to PSR4 but this might involve a mass rename and some tricky scheduling. Let's talk about this during the dev call.

Reply:
Indeed. But for the new things we could opt for using the PSR-4 directly.

Reply:
Yes -- but better than that, we might be able to just rename everything in core code (since we're using classloading) and leave the deprecated `import` function to handle legacy `.inc.php` imports. Needs thought/experimentation.